backend/src/utils/resourceUtils.ts

```diff
@@ -905,6 +910,64 @@ export const migrateTemplateDisablement = async (
     });
 };
 
+/**
+ * As we release Dashboard v2.24.0 we will be re-enabling Trusty for RHOAI, but not for Bias Metrics.
+ * This means we will need to make sure Bias is in a state of disabled while potentially the client
+ * reads Bias as
+ */
+export const migrateBiasTrustyForRHOAI = async (
+  fastify: KubeFastifyInstance,
+  dashboardConfig: DashboardConfig,
+): Promise<DashboardConfig> => {
+  const { namespace, customObjectsApi } = fastify.kube;
+
+  // TODO: There should be a better way to go about this... but the namespace is unlikely to ever change
```

Comment:
Should we just create a constant for `'redhat-ods-applications'` in `backend/src/utils/constants.ts`, or maybe a `isRHOAI` util?

Reply:
It's a valid question -- I feel this is not something I want to encourage... but maybe that's a good use for a utility. I'll go ahead and document the function. Thanks!

Reply:
We used to have something like that for prometheus, I think we remove it in a refactoring.

Also, DSC has a `release` section, detailing wether it's open data hub or rhoai, maybe that's something that we could use to determine wether we are using ODH or rhoai right?